web/angular-wallet/src/app/store/store.service.ts

```diff
@@ -236,4 +236,47 @@ export class StoreService {
             }
         });
     }
+
```

Comment:
Not sure if needed, but I figured storing the blocks in loki might be a good idea.